AD419.DataHelper.Web/Views/Shared/_TopNav.cshtml

```diff
@@ -46,8 +46,9 @@
                 <li class="dropdown">
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">Review <span class="caret"></span></a>
                     <ul class="dropdown-menu">
-                        <li>@Html.ActionLink("FFY Expenses by ARC Report Generation Criteria", "Index", "FFYExpensesByARCReport")</li>
-                        <li>@Html.ActionLink("FFY Expenses by ARC", "Report", "FFYExpensesByARCReport")</li>
+                        <li>@Html.ActionLink("Expenses by ARC/Account Report Generation Criteria", "Index", "FFYExpensesByARCReport")</li>
+                        <li>@Html.ActionLink("FFY Expenses by ARC", "Report/0", "FFYExpensesByARCReport")</li>
+                        <li>@Html.ActionLink("SFY Expenses by ARC", "Report/1", "FFYExpensesByARCReport")</li>

```

Comment:
@taylorkj the second param should be action name, not path, so don't do "Report/0".  You need to pass parameters in the route values, which is the last param.  Like `@Html.ActionLink("text", "action", new { useStateFiscalYear = true })`.  Or for a separate controller `@Html.ActionLink("text", "action", "controller", new { useStateFiscalYear = true }, null)